ddtrace/internal/writer.py

```diff
@@ -133,6 +133,11 @@ def flush_queue(self):
                 self._last_thread_time = new_thread_time
                 self.dogstatsd.histogram('datadog.tracer.writer.cpu_time', diff)
 
+    def _histogram_with_total(self, name, value, tags=None):
+        """Helper to add metric as a histogram and with a `.total` counter"""
+        self.dogstatsd.histogram(name, value, tags=tags)
+        self.dogstatsd.increment('%s.total' % (name, ), value, tags=tags)
```

Comment:
just you need `name` but if you want to decorate it with fancy parentheses, who am I to judge?

Reply:
It is habit because I don't trust that someone won't pass in `name` as a tuple... thank you type checking :p

Reply:
If they pass a tuple, they'll get a `TypeError` which makes them possible to fix their code right away.

Now, if they pass a tuple, they have no error and good luck debugging that…

Reply:
it is called being "robust" 😆